### PR TITLE
Changes to use the opendatahub-operator instead of kfctl.

### PR DESCRIPTION
**Pull request by crobby**
Also adapting to changes in the peak test runner.

### CODE REVIEW

**Comment by crobby** [comment]
Closes #61

File: tests/kubeconfig [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    server: https://api.croberts3.dev.datahub.redhat.com:6443
```
Comment (vpavlin) [inline tests/kubeconfig:5]: @crobby  Did you really want to commit this file?

Comment (crobby) [reply tests/kubeconfig:5]: oh jeez

**Pull request merged**